### ISSUE TITLE
Room Booking Update

### ISSUE THREAD

**Issue by RagerGore018** [5CS024-GROUP-12/AppMockUp#1]
Current state of the room booking is close to full completion.
The current main issues are the following:

-The database doesn't include pictures
-Sometimes the program will crash while updating the booking database

**Comment by Theophilus50** [comment]
yes I am aware pictures weren't added to the database. is there any reason why the program would crash as you said? remember we are meant to ensure it perfect for client usage.

**Comment by RagerGore018** [comment]
The specific way i was trying to implement images in the database(Uploading as bitmaps and then rading said bitmaps when necesary)seems to be stressing out the engine or I have not understood it completly and got something wrong. I have been looking into alteranative solutions

**Comment by cosmin35** [comment]
I wasn't able to actually locate the database, searched through all the files.
